recipe/meta.yaml

```diff
@@ -14,12 +14,13 @@ source:
   '{{ hash_type }}': '{{ hash_value }}'
 
 build:
-  noarch: python
   number: 0
   script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  noarch: python
```

Comment:
Why was the line moved here? I think recipes should be as close as possible to the conda-forge ones, so that we can contribute back to them with minimal efforts/modifications. Please revert this change.